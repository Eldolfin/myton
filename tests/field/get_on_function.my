def foo():
  pass

foo.bar
