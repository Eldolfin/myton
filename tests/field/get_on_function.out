[31merror[error]: 'function' object has no attribute 'bar'
----- Traceback -----
  â”Œâ”€ <unknown>:4:7
  | 
4 | foo.bar
  |        ^
[0m